tools/scripts/getPBOName.bat

```diff
@@ -18,13 +18,13 @@ if [%path%] == [] (
 	goto :end
 )
 
-if not exist "%path%" (
+if not exist %path% (

```

Comment:
Warum hast du hier die Anführungszeichen raus genommen?

Reply:
Weil durch die Parameterübergabe die Anführungszeichen schon in der Variable selbst drinstehen.

Reply:
Alles klar